Example/ViewController.swift

```diff
@@ -20,12 +20,41 @@ class ViewController: UIViewController, GalleryItemsDataSource, GalleryDisplaced
     @IBOutlet weak var image6: UIImageView!
     @IBOutlet weak var image7: UIImageView!
 
-    var imageViews: [UIImageView] = []
+    var imageViews: [UIImageView]              = []
```

Comment:
No problem. I've got AppCode set up to do this for me, but I can turn it off for this.